vignettes/data_protocol.Rmd

```diff
@@ -16,58 +18,49 @@ knitr::opts_chunk$set(
 
 ## About the package
 
-The [`epiparameter` R package](https://github.com/epiverse-trace/epiparameter) contains a library of epidemiological distribution data and functions that read and handle this data. The delay distributions describe the time between two events in epidemiology, for example incubation periods, serial interval and onset-to-death; while the offspring distributions describe the number of secondary infections from a primary infection in disease transmission. The library is compiled by a process of collecting, reviewing and extracting data from peer-reviewed literature[^1], including research articles, systematic reviews and meta-analyses. 
+The [`{epiparameter}` R package](https://github.com/epiverse-trace/epiparameter) contains a library of epidemiological distribution data and functions that read and handle this data. The delay distributions describe the time between two events in epidemiology, for example incubation period, serial interval and onset-to-death; while the offspring distributions describe the number of secondary infections from a primary infection in disease transmission. The library is compiled by a process of collecting, reviewing and extracting data from peer-reviewed literature[^1], including research articles, systematic reviews and meta-analyses. 
 
-To prevent bias in the collection or assessment of the data, a well-defined methodology of searching and refining, just as with other systematic reviews and meta-analyses, is required. This document aims to provide transparency on the methodology used by the `epiparameter` maintainers by outlining the steps taken at each stage of the data handling. This document should also facilitate reproducibility in the searches, results and appraisal steps.
+To prevent bias in the collection or assessment of the data, a well-defined methodology of searching and refining, just as with other systematic reviews and meta-analyses, is required. This document aims to provide transparency on the methodology used by the `{epiparameter}` maintainers by outlining the steps taken at each stage of the data handling. It can also serve as a guide to contributors wanting to search and provide epidemiological parameters currently missing from the library. This document should also facilitate reproducibility in the searches, results and appraisal steps.
```

Comment:
This would be good to add to the 'contributions' blog/website under the 'participation' category

Reply:
The contribution website is not yet live, will link to the contributions to the Epiverse website when available.

Reply:
Link to contributor google sheet (same link as readme).